Dockerfile

```diff
@@ -1,13 +1,24 @@
-# Image: Ubuntu 20.04 Stable, Official Image from Canonical 
+# Image: Ubuntu 20.04 Stable, Official Image from Canonical
 FROM public.ecr.aws/lts/ubuntu:20.04_stable
 
 # Performs updates and installs git, make, curl, python3.8, python3-pip, python3.8-dev and pylint packages
+# Line 13 is required by the spacepy Python package
+# Line >=14 installs cdflib
 RUN apt-get update && \
     apt-get -y upgrade && \
     apt-get -y install git && \
     apt-get -y install make && \
     apt-get -y install curl && \
-    apt-get -y install --no-install-recommends -y python3.8 python3-pip python3.8-dev pylint
+    apt-get -y install wget && \
+    apt-get -y install gfortran && \
+    wget https://spdf.gsfc.nasa.gov/pub/software/cdf/dist/cdf38_0/cdf38_0-dist-cdf.tar.gz && \
+    tar zxvpf cdf38_0-dist-cdf.tar.gz && rm cdf38_0-dist-cdf.tar.gz && \
+    apt-get -y install libncurses5-dev && \
+    apt-get -y install gcc && \
+    cd cdf38_0-dist && \
+    make OS=linux ENV=gnu all && \
+    make INSTALLDIR=/usr/local/cdf install && \
+    cp bin/* /usr/bin/
```

Comment:
There would be a `bin` directory with a `make` command without the `INSTALLDIR` option provided. Given that the binaries get installed in `/usr/local/cdf/bin` couldn't we just add that directory to `PATH` instead of copying them into `/usr/local/bin/`? Not sure how I would do that. Updating `PATH` always confuses me since it depends on which shell you are using.